includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -527,6 +543,14 @@ public function enqueue_scripts() {
 					( ! $settings->is_credit_enabled() ? array( 'credit' ) : array() )
 				);
 				$data['disallowed_methods'] = array_map( 'strtolower', $data['disallowed_methods'] );
+
+				if (
+					! in_array( 'card', $data['disallowed_methods'] )
```

Comment:
Same as the other condition (with the scenarios reversed). I also think it'd be useful to add the comment about hiding credit/debit card options to clarify what's happening here.